cmd/ncproxy/run.go

```diff
@@ -35,13 +36,18 @@ type nodeNetSvcConn struct {
 var (
 	// Global mapping of network namespace ID to shim compute agent ttrpc service.
 	containerIDToShim = make(map[string]computeagent.ComputeAgentService)
+
+	// Global database for containerID to compute agent address
+	agentStore *computeAgentStore
+
 	// Global object representing the connection to the node network service that
 	// ncproxy will be talking to.
 	nodeNetSvcClient *nodeNetSvcConn
 )
 
 var (
 	configPath    = flag.String("config", "", "Path to JSON configuration file.")
+	dbPath        = flag.String("database-path", "", "Path to database storing information on container to compute agent mapping")
```

Comment:
I know we talked offline about whether the log file (and this) should just be a config file option or flags. What made you pick flag (I think either is fine btw 😁)?

Reply:
Config seems specific to the service while flags seem specific to the administration of the service.

Reply:
You've put into words what I was struggling to the other day 😂